src/Inertial_TEST.cc

```diff
@@ -735,3 +739,124 @@ TEST(Inertiald_Test, SubtractionInvalid)
     EXPECT_TRUE((i2 - i1).MassMatrix().IsValid());
   }
 }
+
+TEST(Inertiald_Test, BodyMatrix)
+{
+  math::MassMatrix3d massMatrix(100, {1, 2, 3}, {4, 5, 6});
+  math::Pose3d com{7, 8, 9, 0, 0, 0};
```

Comment:
I would suggest a non-zero pose rotation so we can confirm how we expect that to work (I think it only affects the bottom right sub matrix)